windows-exe/rabbitmq_nsi.in

```diff
@@ -134,7 +134,7 @@ Section "RabbitMQ Server (required)" Rabbit
 
   ; Write the uninstall keys for Windows
   WriteRegStr HKLM ${uninstall} "DisplayName" "RabbitMQ Server %%VERSION%%"
-  WriteRegStr HKLM ${uninstall} "UninstallString" "$INSTDIR\uninstall.exe"
+  WriteRegStr HKLM ${uninstall} "UninstallString" "$\"$INSTDIR\uninstall.exe$\""
```

Comment:
Are you sure about the added '$' characters? Shouldn't it be this instead (but I don't know this language at all so perhaps I'm wrong!)
```suggestion
  WriteRegStr HKLM ${uninstall} "UninstallString" "\"$INSTDIR\uninstall.exe\""
```